central/reports/common/query_builder_test.go

```diff
@@ -66,8 +68,10 @@ var vulnFilters = &storage.VulnerabilityReportFilters{
 }
 
 func TestBuildQuery(t *testing.T) {
-	qb := NewVulnReportQueryBuilder(clusters, namespaces, accessScope, vulnFilters, time.Now())
-	rq, err := qb.BuildQuery()
+	// TODO : Skip is collections is enabled
```

Comment:
Is this task to be completed in this PR? If yes it appears that still needs to be done. If not do we have a JIRA filed for this, and can we include that reference in the comment here?

Reply:
I think that should be done when collections is enabled by default. Created [ROX-13958](https://issues.redhat.com/browse/ROX-13958) and referenced it in the comment.